fccodegen/__init__.py

```diff
@@ -9,19 +9,18 @@
 from ._config import (   # noqa
     DIR_DATA,
     DIR_ROOT,
-    DIR_TEMPLATE,
+    SUBDIR_TEMPLATE,
```

Comment:
'Tis a very minor thing, but my personal preference would be to call these `DATA_DIR`, `ROOT_DIR` and `TEMPLATE_DIR`. I thought for a moment this was a template for subdirectories...

Reply:
Done!